### DISCUSSION TITLE
Subgroups with different identities that can be moved

### DISCUSSION THREAD

**Discussion by Novell2** [dwainscheeren/dwains-lovelace-dashboard#807] [General]
Hello everyone, as a greenhorn I have installed Dwain's Dashboard 3.0. In the video a Walkthrough and First Look he created a subgroup named "All my Devices". But when I create it, my title "Kachel" looks completely different, see printscreen. The goal would be to create subgroups and then move the different entities visually to make it look nicer. However, my existing entities are all fixed and I can't move them to the right place.  
It looks completely different in the video!
What do I wrong???
Best regards
Novell1
[It looks completely different in the video.docx](https://github.com/dwainscheeren/dwains-lovelace-dashboard/files/15127749/It.looks.completely.different.in.the.video.docx)

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
